lib/commands/context.js

```diff
@@ -192,8 +194,11 @@ async function setupExistingChromedriver (chromedriver) {
 }
 
 async function setupNewChromedriver (opts, curDeviceId, adbPort) {
+  let getPort = BlueBird.promisify(PortFinder.getPort, {context: PortFinder});
+  let chromePort = await getPort();
+  logger.error(`Using ${chromePort} for new ChromeDriver session`);
   let chromeArgs = {
-    port: opts.chromeDriverPort,
+    port: chromePort,
```

Comment:
ok, sorry I was confused, I thought this was the `appium-chromedriver` package :-)

I'm a little conflicted because really in the future all this logic will live in the `driverData` section where a new driver will ensure it is not conflicting in terms of ports or other system resources with other drivers. but we can refactor that out.

Also, `--chromedriver-port` is a server flag in Appium that maps to the `chromeDriverPort` capability, and is probably how `opts.chromeDriverPort` gets populated. So I do think we want to respect it if it exists. Unfortunately it might always exist as a default...